packages/ui-scripts/lib/test/lint.js

```diff
@@ -51,7 +51,8 @@ export default {
     if (jspaths.length) {
       commands['eslint'] = getCommand('eslint', [
         ...jspaths,
-        '--ext .js,.jsx,.ts,.tsx',
+        '--ext',
+        '.js,.jsx,.ts,.tsx',
```

Comment:
This was causing an error when running `lint`, I have no idea why it did not came up until now.